tests/test_agents.py

```diff
@@ -433,6 +434,22 @@ async def test_agent_sharing_state(
     summary_llm_model = agent_test_settings.get_summary_llm()
     embedding_model = agent_test_settings.get_embedding_model()
 
+    callbacks = {}
+    if callback_type == "async":
+        gen_answer_initialized_callback = AsyncMock()
+        gen_answer_completed_callback = AsyncMock()
+        gather_evidence_initialized_callback = AsyncMock()
+        gather_evidence_completed_callback = AsyncMock()
+
+        callbacks = {
```

Comment:
Not sure it really matters, but it's more future proof to use `callbacks.update` here instead of redefining